csharp/Worker/Microsoft.Spark.CSharp/Worker.cs

```diff
@@ -36,8 +36,9 @@ public class Worker
         static void Main(string[] args)
         {
             PrintFiles();
-            int javaPort = int.Parse(Console.ReadLine());
```

Comment:
This is not an interactive scenario. Port number is written to output stream by https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/api/python/PythonWorkerFactory.scala#L125 and read here.